pom.xml

```diff
@@ -155,6 +159,27 @@
       <artifactId>bigquery-connector</artifactId>
       <version>${bigquery.connector.hadoop2.version}</version>
     </dependency>
+    <dependency>
+      <groupId>com.google.cloud.bigtable</groupId>
+      <artifactId>bigtable-hbase-1.x-mapreduce</artifactId>
+      <version>${google.cloud.bigtable.version}</version>
+      <exclusions>
+        <exclusion>
+          <groupId>org.apache.hbase</groupId>
+          <artifactId>hbase-server</artifactId>
+        </exclusion>
+      </exclusions>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.hbase</groupId>
+      <artifactId>hbase-shaded-server</artifactId>
+      <version>${hbase-shaded-server.version}</version>
+    </dependency>
+    <dependency>
```

Comment:
can you add a comment about what requires this dependency?

Reply:
added